### PR TITLE
Added new Unique Tokens (ERC721)  View integration with OpenSea API . Closes #240.

### LINKED ISSUE #240: 😻 Unique Tokens (Nerds call them Non-Fungible Tokens)
We need to display our customers' kitties.

### PR DESCRIPTION
**Pull request by palevoo**
### PR Checklist (check all)

* [x] Updated `CHANGELOG.md` to describe the included fixes and changes made
* [x] Included issue number on the description of the PR
* [x] Commented on the relevant issue thread about this PR

### Issue number
240

### Changelog
Added support for ERC721 from OpenSea API visualised in the new account view

### CODE REVIEW

**Comment by ricburton** [comment]
🙌 Preview this pull request.

Built with commit ec794c6388fd24ff3c9b98ba0e6ccf27ca34b7f5

https://deploy-preview-266--balance-manager.netlify.com

**Comment by ricburton** [comment]
This is great!

@palevoo Did you get the new link/token from OpenSea for this to avoid CORS?

(@jinchung need to check that before we merge.)

**Review by jinchung — COMMENTED** [review]
_uniqueitems.js and erc721.json are no longer needed correct?

File: src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    margin-left: 15px;
+  }
+`;
+
+const Footer = () => (
+  <Container>
+    <TextField>
+      <Header>{lang.t('message.opensea_header')}</Header>
+      <Text>
+        <Link href="https://opensea.io/">OpenSea</Link>
```
Comment (jinchung) [inline src/components/Footer.js]: Could we make both of these links open to a new window

**Comment by palevoo** [comment]
correct, not needed anymore, apologies

File: src/views/Account/AccountUniqueTokens.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          </StyledContainer>
+          <Footer />
+        </UniqueTokensContainer>
+      ) : (
+        <StyledCard minHeight={280} fetching={this.props.fetchingTransactions}>
+          <StyledMessage>{lang.t('message.failed_request')}</StyledMessage>
+        </StyledCard>
+      )
+    ) : (
+      <StyledCard minHeight={280} fetching={this.props.fetchingTransactions}>
```
Comment (jinchung) [inline src/views/Account/AccountUniqueTokens.js]: I think you mean to be doing this based on "this.props.fetchingUniqueTokens" (same with line 51)

**Review by jinchung — COMMENTED** [review]
@palevoo Thanks for all your work! Do you know if OpenSea has a CORS enabled endpoint? ah wait - I see we have an update with an API key - cheers!

**Review by jinchung — APPROVED** [review]

**Pull request merged**